bibliopixel/drivers/serial_driver.py

```diff
@@ -248,13 +257,12 @@ def _generateHeader(cmd, size):
         packet.append(size >> 8)
         return packet
 
-    @staticmethod
-    def setDeviceID(dev, id):
+    def setDeviceID(self, dev, id):
         if id < 0 or id > 255:
             raise ValueError("ID must be an unsigned byte!")
 
         try:
-            com = serial.Serial(dev, timeout=5)
+            com = serial.Serial(dev, baudrate=self.baudrate, timeout=5)
```

Comment:
If we're going to bring in the baudrate as a parameter, we should also bring in the timeout as well conceivably.